lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,14 +35,24 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
-            View::Template.new(path, @view.configuration.default_encoding)
-          else
-            super
-          end
+          find_cached_template
         end
 
         protected
+
+        # @since x.x.x
+        # @api private
+        def find_cached_template
+          conf = Lotus::View::Configuration.for(@view)
```

Comment:
@jodosha using `@view.configuration` doesn't work for me. This area is still confusing me but the partials are being attached to the configuration object on the application namespace and not being copied down to the configurations for the individual views. I see why this is expensive but not sure how to resolve it in a more efficient way.

I think I've fixed all the other issues you've raised.